src/component/Chat/SideBar/SideBar.js

```diff
@@ -0,0 +1,16 @@
+// Sidebar.js
+import "./SideBar.css"; // Import the CSS file
+const SideBar = ({ Users }) => {
+  return (
+    <div className="sidebar">
+      <h2>Users</h2>
+      <ul>
+        {Users.map((user, index) => (
+          <li key={index}>{user.user_name}</li>
```

Comment:
React recommends that you do not use indexes as keys, since it could impact performance negatively and could lead to some unstable component behavior. You can user user id as key

Reply:
Done this request.